docs/examples.md

```diff
@@ -0,0 +1,99 @@
+
+# Textbook Examples
+
+A significant examples directory attached to cirq contains many useful
+examples that you can use to see how to write different algorithms.
+Most of these can be found in the
+[examples directory](https://github.com/quantumlib/Cirq/tree/master/examples)
+in the Github repository.
+
+In addition to these examples, you can also find the notebooks that generated
+the cirq docs found here in the
+[docs directory](https://github.com/quantumlib/Cirq/tree/master/docs) as well
+as all the more advanced case studies listed below this page in the table of
+contents.
+
+Below is a listing of the examples in the repository:
+
+## Beginning algorithms
+
+*    [Hello Qubit](https://github.com/quantumlib/Cirq/blob/master/examples/hello_qubit.py)
+Simple first program showing how to create a circuit.
+
+*    [Deutsch Algorithm](https://github.com/quantumlib/Cirq/blob/master/examples/deutsch.py)
+Textbook example of the simplest quantum advantage.
+
+*    [Bernstein-Vazirani](https://github.com/quantumlib/Cirq/blob/master/examples/bernstein_vazirani.py)
+Textbook algorithm determining properties of a hidden "oracle" function
```

Comment:
Suggestion: "Textbook algorithm determining a global property of a function with surprisingly few calls to it."

Reply:
Done.